### ISSUE TITLE
config file

### ISSUE THREAD

**Issue by FernandoMiguel** [awslabs/eks-node-viewer#33]
can this read a config file so we store regular parameters?
`--extra-labels=karpenter.sh/provisioner-name,topology.kubernetes.io/zone -resources memory,cpu,pods`
if there was a `$HOME/.eks_node_viewer` with the contents:
```ini
extra-labels=karpenter.sh/provisioner-name,topology.kubernetes.io/zone 
resources=memory,cpu,pods
```

**Comment by pete911** [comment]
Hi @FernandoMiguel, I have created PR to address this - https://github.com/awslabs/eks-node-viewer/pull/34

**Comment by tzneal** [comment]
Fixed in #34 , thanks @pete911
